src/test/java/tests/TurboMilestoneTest.java

```diff
@@ -34,6 +32,49 @@ public void turboMilestoneTest() {
+        // milestone without due date
+        TurboMilestone milestoneWithDueDateClosed1 = new TurboMilestone("1", 1, "milestone01");
+        milestoneWithDueDateClosed1.setDueDate(Optional.of(LocalDate.now().plusDays(5)));
+        milestoneWithDueDateClosed1.setOpen(false);
+        TurboMilestone milestoneWithDueDateClosed2 = new TurboMilestone("2", 2, "milestone02");
+        milestoneWithDueDateClosed2.setDueDate(Optional.of(LocalDate.now().plusDays(3)));
+        milestoneWithDueDateClosed2.setOpen(false);
+
+        TurboMilestone milestoneWithDueDateOpen1 = new TurboMilestone("1", 11, "milestone21");
+        milestoneWithDueDateOpen1.setDueDate(Optional.of(LocalDate.now().plusDays(2)));
+        milestoneWithDueDateOpen1.setOpen(true);
+        TurboMilestone milestoneWithDueDateOpen2 = new TurboMilestone("1", 12, "milestone22");
+        milestoneWithDueDateOpen2.setDueDate(Optional.of(LocalDate.now().plusDays(4)));
+        milestoneWithDueDateOpen2.setOpen(true);
+
+        TurboMilestone milestoneWithoutDueDateOpen1 = new TurboMilestone("1", 21, "milestone31");
+        TurboMilestone milestoneWithoutDueDateOpen2 = new TurboMilestone("2", 21, "milestone32");
+
+        TurboMilestone milestoneWithoutDueDateClosed1 = new TurboMilestone("1", 31, "milestone11");
+        milestoneWithoutDueDateClosed1.setOpen(false);
+        TurboMilestone milestoneWithoutDueDateClosed2 = new TurboMilestone("1", 32, "milestone12");
+        milestoneWithoutDueDateClosed2.setOpen(false);
+
+        Comparator<TurboMilestone> comparator = TurboMilestone.getDueDateComparator();
+        List<TurboMilestone> milestones = Arrays.asList(milestoneWithDueDateOpen1, milestoneWithDueDateOpen2,
+                                                        milestoneWithDueDateClosed1, milestoneWithDueDateClosed2,
+                                                        milestoneWithoutDueDateOpen1, milestoneWithoutDueDateOpen2,
+                                                        milestoneWithoutDueDateClosed1, milestoneWithoutDueDateClosed2);
+        Collections.sort(milestones, comparator);
+        List<TurboMilestone> expectedMilestoneList = Arrays.asList(milestoneWithoutDueDateClosed1,
+                                                                    milestoneWithoutDueDateClosed2,
+                                                                    milestoneWithDueDateOpen1,
+                                                                    milestoneWithDueDateClosed2,
+                                                                    milestoneWithDueDateOpen2,
+                                                                    milestoneWithDueDateClosed1,
+                                                                    milestoneWithoutDueDateOpen1,
+                                                                    milestoneWithoutDueDateOpen2);
```

Comment:
Sorry, I missed this part. Look at the indentation of the test above (aligned with `(`). In fact, you can pack 2 parameter per line if it fits.